utils/workshop_utils.py

```
@@ -0,0 +1,242 @@
+        print(error_msg)
+        # 尝试使用logger记录错误
+        try:
+            import logging
+            logger = logging.getLogger(__name__)
+            logger.error(error_msg)
+        except Exception:
+            pass
+
+def ensure_workshop_folder_exists(folder_path: Optional[str] = None) -> bool:
+    """
+    确保创意工坊文件夹存在，如果不存在则自动创建
```

Ruff (0.14.5)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)